tools/PrepareRelease/Program.cs

```diff
@@ -0,0 +1,41 @@
+using System.Linq;
+using Castle.Core.Internal;
+
+namespace PrepareRelease
+{
+    public class Program
+    {
+        public const string Versions = "versions";
+        public const string Integrations = "integrations";
+        public const string Msi = "msi";
+        public const string AzureAppServices = "aas";
+
+        public static void Main(string[] args)
+        {
+            if (JobShouldRun(Versions, args))
+            {
+                SetAllVersions.Run();
+            }
+
+            if (JobShouldRun(Integrations, args))
+            {
+                GenerateIntegrationDefinitions.Run();
+            }
+
+            if (JobShouldRun(Msi, args))
+            {
+                SyncMsiContent.Run();
+            }
+
+            if (JobShouldRun(AzureAppServices, args))
+            {
+                SyncAzureAppServicesPackage.Run();
+            }
+        }
+
+        private static bool JobShouldRun(string jobName, string[] args)
+        {
+            return args.IsNullOrEmpty() || args.Any(a => a.ToLower().Equals(jobName));
```

Comment:
I know it's an internal tool, but Best Practices!
```suggestion
            return args.IsNullOrEmpty() || args.Any(a => string.Equals(a, jobName, StringComparison.OrdinalIgnoreCase);
```